react/lib/constants.ts

```diff
@@ -0,0 +1,33 @@
+import { GridStackOptions } from 'gridstack';
+
+export const BREAKPOINTS = [
+  { c: 1, w: 700 },
+  { c: 3, w: 850 },
+  { c: 6, w: 950 },
+  { c: 8, w: 1100 },
+];
+
+export const SUB_GRID_OPTIONS: GridStackOptions = {
+  acceptWidgets: true,
+  cellHeight: 54,
+  column: 'auto',
+  columnOpts: {
+    breakpoints: BREAKPOINTS,
```

Comment:
nested grid either use column:'auto' (get size from parent) OR breakpoint, not both... I use auto so dragging in/out keeps items the same which is more common.

Reply:
humm, again I would do `column:'auto'` for nested. you happen to have a section take entire width of the grid so you are lucking out that parent & child matches, but this would break if it wasn't full width.
we can update later...